plugins/inputs/docker/docker.go

```diff
@@ -56,6 +57,9 @@ var sampleConfig = `
   endpoint = "unix:///var/run/docker.sock"
   ## Only collect metrics for these containers, collect all if empty
   container_names = []
+  ## Only collect these container labels from docker daemon, collect all if empty but note that
```

Comment:
please remove the mention of the prometheus output, this is too specific for an option that is in fact general.